src/helpers/figures/charts/chart_common.ts

```diff
@@ -428,3 +437,80 @@ export function getDefinedAxis(definition: ChartWithAxisDefinition): {
   useLeftAxis ||= !useRightAxis;
   return { useLeftAxis, useRightAxis };
 }
+
+export function getTrendDatasetForBarChart(config: TrendConfiguration, dataset: any) {
+  const values: number[] = [];
+  const labels: number[] = [];
+  for (let i = 0; i < dataset.data.length; i++) {
+    values.push(dataset.data[i]);
+    labels.push(i + 1);
+  }
```

Comment:
Is there any different behavior between this and line charts with `axisType` `"category"` ? could the same code be re-used ?

Reply:
Yes, there is a difference for the axis. For 10 values, the bar chart axis will go from 0.5 to 10.5, while the line chart will go from 1 to 10.